modules/rostests/apitests/imm32/clientimc.c

```diff
@@ -0,0 +1,46 @@
+/*
+ * PROJECT:     ReactOS api tests
+ * LICENSE:     GPL-2.0-or-later (https://spdx.org/licenses/GPL-2.0-or-later)
+ * PURPOSE:     Test for imm32 ImmLockClientImc/ImmUnlockClientImc
+ * COPYRIGHT:   Copyright 2021 Katayama Hirofumi MZ (katayama.hirofumi.mz@gmail.com)
+ */
+
+#include <ntstatus.h>
+#define WIN32_NO_STATUS
+#include <windows.h>
+#include <imm.h>
+#include <ddk/imm.h>
```

Comment:
ddk?

Reply:
Yeah, Driver Development Kit (DDK) had used to have header file `immdev.h` to develop IMEs.
https://github.com/reactos/reactos/blob/master/sdk/include/reactos/wine/ddk/imm.h

Reply:
I see... then maybe sometime in the future this wine header will need to be converted into something MS-compatible that we can put in our ddk...